requirements.txt

```diff
@@ -1,10 +1,9 @@
 PyStemmer>=1.3,<2.0
-tensorflow>=1.5,<2.0
 bblfsh>=2.2.1,<3.0
 modelforge>=0.5.4-alpha,<0.6
-sourced-engine>=0.5.1,<0.6
+sourced-engine>=0.5.1,<0.7
 parquet>=1.2,<2.0
 numpy>=1.14
 humanize>=0.5.0,<0.6
-pyspark>=2.2.0,<2.2.1
 pygments>=2.2.0,<3.0
+tensorflow>=1.5,<2.0
```

Comment:
We do not include tensorflow into the requirements: there can be `tensorflow-gpu` too.

Reply:
okay done